application/views/template.php

```diff
@@ -1,17 +1,42 @@
-<!DOCTYPE html>
-<html lang="en">
-	<head>
-		<meta charset="utf-8">
-		<title>{pagetitle}</title>
+<?php
+if (!defined('APPPATH'))
+	exit('No direct script access allowed');
+/**
+ * views/template.php
+ *
+ * Pass in $pagetitle (which will in turn be passed along)
+ * and $pagebody, the name of the content view.
+ *
+ * Yes, this is using an old version of Bootstrap. too bad.
+ *
+ * ------------------------------------------------------------------------
+ */
+?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <title>{pagetitle}</title>
         <meta HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=UTF-8"/>
         <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-        <link rel="stylesheet" type="text/css" href="/assets/css/default.css"/>
```

Comment:
i should prob remove the default. will do in this branch if further changes are required otherwise will do in another